Rubberduck.Core/Navigation/CodeExplorer/CodeExplorerMemberViewModel.cs

```diff
@@ -9,9 +9,9 @@ namespace Rubberduck.Navigation.CodeExplorer
 {
     public class CodeExplorerMemberViewModel : CodeExplorerItemViewModel
     {
-        public CodeExplorerMemberViewModel(ICodeExplorerNode parent, Declaration declaration, IEnumerable<Declaration> declarations) : base(parent, declaration)
+        public CodeExplorerMemberViewModel(ICodeExplorerNode parent, Declaration declaration, List<Declaration> declarations) : base(parent, declaration)
```

Comment:
Does it have to be `List`? Can't it be `IList`?

Reply:
Maybe? Changing that to use the interface would require changing it everywhere else in the object hierarchy (including on the interface) though, so I'm not horribly inclined to change it.  Given that it's fairly standard to call `ToList()` on an `IEnumerable` to materialize it, using the strong type makes a certain amount of sense.